doc/tutorials/notifications.md

```diff
@@ -53,125 +47,163 @@ notifications but not whether notifications are sent or which transport is used.
 How notifications are sent is specified in the *notification configuration* by
 the system operator.
 
-## Notification configuration
+## Notification Configuration
 
 System operators can mount the desired transport plugins and configure them
-(e.g. set channel, host, port and credentials) either globally or when mounting
-a configuration file.
+(e.g. set channel, host, port and credentials) globally.
 
 They need to mount both sending and receiving plugins in order to use a
 transport.
 
 ```sh
-kdb mount file.ini system/example ini dbussend dbusreceive
+kdb global-mount dbus announce=once dbusrecv
 ```
 
+Plugins usable as transport plugin are marked with `notification` on the
+[plugin page](https://www.libelektra.org/plugins/readme#notification-and-logging).
+
 ## How to integrate an I/O binding and send notifications asynchronously
 
 Developers do not need to change their programs in order to start sending
 notifications.
-However without the integration of an I/O binding notifications are sent
-synchronously which will block normal program execution.
+However without the integration of an I/O binding notifications *may* be sent
+synchronously which would block normal program execution.
 For programs without time constraints (e.g. CLI programs) this may not be
 important, but for GUIs or network services this will have negative impact.
 
-Since many different I/O management libraries exist (e.g. libuv, glib or libev)
-the transport plugins use the I/O interface for their I/O operations.
-Each I/O management library needs its own I/O binding.
-Developers can also create their own I/O binding for the I/O management library
-of their choice.
-This is described in the last section.
+The "zeromqsend" and "dbus" plugins do not block program execution for sending
+as sending is handled asynchronously be the underlying libraries.
 
-Each I/O binding has its own initialization function that creates a new
-I/O binding and connects it to the I/O management library.
-For this tutorial we will assume that libuv is used.
-For details on how to use a specific binding please look at the bindings'
-README.md in `src/bindings/io/<binding>`.
+The following is a basic example of an application using Elektra extended by
+the initialization of an I/O binding.
 
 ```C
 #include <elektra/kdb.h>
 #include <elektra/kdbio.h>
-#include <elektra/kdbio_uv.h>
-#include <elektra/kdbnotification.h>
+#include <elektra/kdbio/uv.h>
 
 #include <uv.h>
 
 void main (void)
 {
-	KDB* repo;
+  KDB* repo;
```

Comment:
Any specific reason why the change to spaces here?

Reply:
No, no reason. Might be the editor messing with identation or me forgetting about it when pasting code.